test/remote/gateways/remote_orbital_test.rb

```diff
@@ -217,6 +220,104 @@ def test_successful_purchase_with_discover_network_tokenization_credit_card
     assert_false response.authorization.blank?
   end
 
+  # Successful purchase through Electronic Check
```

Comment:
These function level comments aren't necessary since the code is generally self documenting.

Reply:
The idea was to provide a single line description of the tests in plain text. If required, these can be removed.

Reply:
Yes, these comments are not needed.

Reply:
Acknowledged. We will remove the comments.